lib/plugins/create/create.test.js

```diff
@@ -387,6 +387,38 @@ describe('Create', () => {
       });
     });
 
+    it('should generate scaffolding for "kubeless-python" template', () => {
+      process.chdir(tmpDir);
+      create.options.template = 'kubeless-python';
+
+      return create.create().then(() => {
+        expect(create.serverless.utils.fileExistsSync(path.join(tmpDir, 'package.json')))
+          .to.be.equal(true);
+        expect(create.serverless.utils.fileExistsSync(path.join(tmpDir, 'serverless.yml')))
+          .to.be.equal(true);
+        expect(create.serverless.utils.fileExistsSync(path.join(tmpDir, 'handler.py')))
+          .to.be.equal(true);
+        expect(create.serverless.utils.fileExistsSync(path.join(tmpDir, '.gitignore')))
+          .to.be.equal(true);
+      });
+    });
+
+    it('should generate scaffolding for "kubeless-nodejs" template', () => {
+      process.chdir(tmpDir);
+      create.options.template = 'kubeless-nodejs';
+
+      return create.create().then(() => {
+        expect(create.serverless.utils.fileExistsSync(path.join(tmpDir, 'package.json')))
+          .to.be.equal(true);
+        expect(create.serverless.utils.fileExistsSync(path.join(tmpDir, 'serverless.yml')))
+          .to.be.equal(true);
+        expect(create.serverless.utils.fileExistsSync(path.join(tmpDir, 'handler.js')))
+          .to.be.equal(true);
+        expect(create.serverless.utils.fileExistsSync(path.join(tmpDir, '.gitignore')))
```

Comment:
This file seems to be missing. Please note the comment about the `gitignore` file name below when adding it.